docs/howtos/modules/deadlines/pages/testing.adoc

```diff
@@ -0,0 +1,36 @@
+= Testing the deadline
+
+Testing behaviour related to deadlines is easy by using either the `SagaTestFixture` or the `AggregateTestFixture` these will include configuration to use a `StubDeadlineManager`.
+This makes it possible to simulate passing of time, and checking the behaviour.
+
+[source,java]
+----
+class ExpiringGiftCardTest {
+
+    private static final String CARD_ID = UUID.randomUUID().toString();
+    private static final int DAYS = 30;
+    private static final int AMOUNT = 1377;
+
+    private AggregateTestFixture<ExpiringGiftCard> testFixture;
+
+    @BeforeEach
+    public void setUp() {
+        testFixture = getTestFixture();
+    }
+
+    @Test
+    void testCardExpiredAfterThirtyOneDays() {
+        testFixture.givenNoPriorActivity()
+                   .andGivenCommands(new IssueExpiringCardCommand(CARD_ID, DAYS, AMOUNT))
+                   .whenTimeElapses(Duration.ofDays(31L)) // <1>
+                   .expectSuccessfulHandlerExecution()
```

Comment:
Curious: Why not include an `expectTriggeredDeadlines` validation here?

Reply:
No particular reason. It's a good addition.